chapters/3-package-information.md

```diff
@@ -1047,6 +1047,38 @@ Example:
         </spdx:externalRef>
         ...
     </spdx:package>
+    
+## 3.23 Package Attribution Text <a name="3.23"></a>
+
+**4.15.1** Purpose: This field provides a place for the SPDX data creator to record all attributions at the package level that are required to be communicated. These typically include copyright statement(s), license text, and a disclaimer.
+
+**4.15.2** Intent: The intent is to provide the recipient of the SPDX file with all the legally required attributions at a package level, therefore complying with the license obligations.
+
+**4.15.3** Cardinality: Optional, one or many.
```

Comment:
I think the many cardinality is fine, but we could define this to be optional 1 only and require the text field to contain all necessary attribution text in a single field.

Reply:
It was originally 1,  but I could see where tools might be picking up fragments from various contributors under different licenses in same file which might be awkward to collapse together, so figured multiple didn't hurt anything.